src/schemas/template.ts

```diff
@@ -12,7 +12,7 @@ export const typeDefs = gql`
     "Create a new Template. Leave the 'copyFromTemplateId' blank to create a new template from scratch"
     addTemplate(name: String!, copyFromTemplateId: Int): Template
     "Update a Template"
-    updateTemplate(templateId: Int!, name: String!, visibility: TemplateVisibility!, bestPractice: Boolean): Template
+    updateTemplate(templateId: Int!, name: String!, visibility: TemplateVisibility!, bestPractice: Boolean!): Template
```

Comment:
I think it would be better to just update [this line](https://github.com/CDLUC3/dmsp_backend_prototype/blob/b43496f55b494f1aae203713a4038432d7000beb/src/resolvers/template.ts#L151) instead to ensure that the value for `bestPractice` is set. Only a Super Admin should be able to set this.